docs/blog/oras-0.15-a-fully-functional-registry-client.md

```diff
@@ -0,0 +1,189 @@
+# ORAS 0.15: A fully Functional OCI Registry Client
+
+_Author: [Feynman Zhou](https://twitter.com/FeynmanZhou), [Yi Zha](https://github.com/yizha1)_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced two releases of v0.15 for the ORAS CLI recently. ORAS [v0.15.0](https://github.com/oras-project/oras/releases/tag/v0.15.0) introduces four new top-level commands and new options to manage tags and repositories in registries and provides capabilities to alter the OCI content for advanced use cases. Three weeks later, [ORAS 0.15.1](https://github.com/oras-project/oras/releases/tag/v0.15.1) also released with a few known bug fixes. Since the release of v0.15, ORAS CLI has evolved into a fully functional OCI registry client.
+
+## What's new in ORAS 0.15
+
+![ What's new in ORAS 0.15](oras-0.15/what's-new-in-oras-0.15.png)
+
+As ORAS has been adopted by more and more OCI implementors and registry vendors, we have seen increased community requirements in providing fine-grained capabilities to alter the content of OCI supply chain artifacts. ORAS 0.15 now supports operate the blob and manifest of an artifact in the registry. Please see the [Release Notes](https://github.com/oras-project/oras/releases/tag/v0.15.0) for details.
+
+This blog post will demonstrate how to use ORAS CLI v0.15 to convert a Docker image stored in Docker Hub into an OCI image，then push it to the Distribution registry.
+
+## Run a local Distribution registry
+
+Run a local instance of the CNCF Distribution Registry, with ORAS Artifacts support:
+
+```
+docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v1.0.0-rc.2
+```
+
+## Fetch and view the manifest of a sample Docker image
+
+```
+$ oras manifest fetch docker.io/library/hello-world@sha256:f54a58bc1aac5ea1a25d796ae155dc228b3f0e11d046ae276b39c4bf2f13d8c4
+
+{
+   "schemaVersion": 2,
+   "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+   "config": {
+      "mediaType": "application/vnd.docker.container.image.v1+json",
+      "size": 1469,
+      "digest": "sha256:feb5d9fea6a5e9606aa995e879d862b825965ba48de054caab5ef356dc6b3412"
+   },
+   "layers": [
+      {
+         "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
+         "size": 2479,
+         "digest": "sha256:2db29710123e3e53a794f2694094b9b4338aa9ee5c40b930cb8063a1be392c54"
+      }
+   ]
+}
+```
+
+## Fetch and push a blob
+
+Fetch a config blob to a local file from a sample Docker image:
```

Comment:
Can we use `jq` to avoid mentioning digests in the sample commands? Like:

## Fetch and view the manifest of a sample Docker image
```shell
docker_digest="sha256:f54a58bc1aac5ea1a25d796ae155dc228b3f0e11d046ae276b39c4bf2f13d8c4"
oras manifest fetch docker.io/library/hello-world@$docker_digest > docker.manifest.json
cat docker.manifest.json
{
  "schemaVersion": 2,
  "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
  "config": {
    "mediaType": "application/vnd.docker.container.image.v1+json",
    "size": 1469,
    "digest": "sha256:feb5d9fea6a5e9606aa995e879d862b825965ba48de054caab5ef356dc6b3412"
  },
  "layers": [
    {
      "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
      "size": 2479,
      "digest": "sha256:2db29710123e3e53a794f2694094b9b4338aa9ee5c40b930cb8063a1be392c54"
    }
  ]
}
```
## Fetch and push a blob

Fetch a config blob to a local file from a sample Docker image:

```
config_digest=$(cat docker.manifest.json | jq -r .config.digest)
oras blob fetch docker.io/library/hello-world@$config_digest --output config-blob.json
```

Then push this blob file to a new repository in the Distribution registry:

```
oras blob push localhost:5000/oras-distribution/hello-world config-blob.json
```

Similarly, fetch the layer blob and push it to the Distribution registry:

```
layer_digest=$(cat docker.manifest.json | jq -r .layers[].digest)
oras blob fetch docker.io/library/hello-world@$layer_digest --output layer-blob.json
oras blob push localhost:5000/oras-distribution/hello-world layer-blob.json
```